### PR TITLE
Manage collections by bucket

### PR DESCRIPTION
**Pull request by leplatrem**
Based on #41

### CODE REVIEW

**Comment by leplatrem** [comment]
@Natim r?

Especially one point : 

For records, is ``bucket_id`` the parent_id of records ? and ``collection_id`` the ``collection id`` as it was implemented here ?

Or ``collection_id`` is the ``parent_id``, and ``records`` is the ``collection_id`` (i.e. the object type) ?

**Comment by Natim** [comment]
Following our conversation, I think we have:

- ``parent_id`` which is the ``object_uri`` of the parent ie: ``/buckets/<bucket_id/collections/<collection_id>``
- ``collection_id`` should be ``resource_name`` or ``object_type`` and contains the object type ie: ``record``, ``collection``, ``bucket``

File: kinto/views/records.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                 "/collections/{{collection_id}}/records/{{id}}"),
+    collection_path=("/buckets/{{bucket_id}}"
+                     "/collections/{{collection_id}}/records"))
+class Record(resource.BaseResource):
+
+    mapping = RecordSchema()
+
+    def __init__(self, *args, **kwargs):
+        super(Record, self).__init__(*args, **kwargs)
+        self.collection.collection_id = self.request.matchdict['collection_id']
```
Comment (Natim) [inline kinto/views/records.py]: ``self.collection.collection_id = 'record'``

Comment (Natim) [inline kinto/views/records.py]: Maybe can we get it from the ``resource_name`` directly?

**Comment by leplatrem** [comment]
Thanks @Natim for your feedback, I did these in #48 to avoid merging hell :)

**Pull request merged**